pkg/apis/openebs.io/device/v1alpha1/devicevolume.go

```diff
@@ -77,7 +77,8 @@ type VolStatus struct {
 	// The state "Pending" means that the volume creation request has not
 	// processed yet. The state "Ready" means that the volume has been created
 	// and it is ready for the use.
-	// +kubebuilder:validation:Enum=Pending;Ready
+	// Failed
```

Comment:
nit: you can remove the line `// Failed`.

Reply:
resolved